### PR TITLE
small linter fixes

### PR DESCRIPTION
**Pull request by jenschude**
#### Summary

Fixes the broken linting on CircleCI

#### Todo

- Tests
  - [ ] Unit
  - [ ] Integration
  - [ ] Acceptance
- [ ] Documentation
- [ ] `Type` label for the PR

### CODE REVIEW

File: packages/sdk-middleware-http/src/http.js [hunk 1]
```diff
@@ -120,7 +120,7 @@ export default function createHttpMiddleware({
     if (credentialsMode) {
       fetchOptions.credentials = credentialsMode
     }
-    if (timeout || abortController) {
+    if (abortController) {
```
Comment (jenschude) [inline packages/sdk-middleware-http/src/http.js:123]: FYI: if timeout is set, the abortController will be initialized anyway

Comment (daern91) [reply packages/sdk-middleware-http/src/http.js:123]: Good catch 👍

**Review by daern91 — APPROVED** [review]
LGTM! Thanks a lot for the quick fix 🥇

File: packages/sdk-middleware-http/src/http.js [hunk 2]
```diff
@@ -92,7 +92,7 @@ export default function createHttpMiddleware({
     request: MiddlewareRequest,
     response: MiddlewareResponse
   ) => {
-    let abortController
+    let abortController: any
```
Comment (katmatt) [inline packages/sdk-middleware-http/src/http.js:95]: This is TypeScript and I'm wondering why JavaScript accepts it at all?

Comment (jenschude) [reply packages/sdk-middleware-http/src/http.js:95]: Cause the node SDK uses flow syntax ;)

**Pull request merged**